guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -9,7 +9,7 @@ All subscription information is available in your Red Hat Customer Portal accoun
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat Subscription Manifest in the Customer Portal.
 
 ifdef::satellite[]
-Note that the subscription model is deprecated and will be removed in a future release.
+Note that legacy subscription management, including the need to attach subscriptions to hosts and activation keys, is deprecated and will be removed in a future release.
```

Comment:
Thanks, change looks good me!
Just a tiny detail.
```suggestion
Note that the legacy subscription management, including the need to attach subscriptions to hosts and activation keys, is deprecated and will be removed in a future release.
```

Reply:
No need for a definite article there. "Legacy" is an adjective modifying "subscription management."

I would also be fine with "entitlement-based subscription management" if you like that better; I think I've seen that term elsewhere.

Reply:
Understood. Leaving the current form is fine.